File: challenges/helloworldhelloworld-mcb2hqi6/TestRunner.java
/**
 * Test Runner für HelloworldHelloworld
 */
public class TestRunner {
    public static void main(String[] args) {
        System.out.println("🎯 HelloworldHelloworld");
        System.out.println("Führe Tests aus...\n");
        
        try {
            Tests.runAllTests();
        } catch (Exception ex) {
            System.out.println("❌ Fehler: " + ex.getMessage());
            System.exit(1);
        }
        
        System.out.println("\nTests abgeschlossen.");
    }
}

File: challenges/hal-mcazfk9x/TestRunner.cs
using System;

public class TestRunner
{
    public static void Main(string[] args)
    {
        Console.WriteLine("🎯 hAL");
        Console.WriteLine("Führe Tests aus...\n");
        
        try
        {
            // ✅ KORRIGIERT: Verwendet standardisierten Klassennamen
            Tests.RunAllTests();
        }
        catch (Exception ex)
        {
            Console.WriteLine($"❌ Fehler: {ex.Message}");
            Environment.Exit(1);
        }
        
        Console.WriteLine("\nTests abgeschlossen.");
    }
}